fetch.bs

```diff
@@ -5685,7 +5686,14 @@ that RFC's normative processing requirements. [[RFC2397]]
 
  <li><p>Set <var>mimeTypeBytes</var> to the <a>isomorphic encode</a> of <var>mimeType</var>.
 
- <li><p>Return <var>mimeTypeBytes</var> and <var>body</var>.
+ <li><p>Let <var>dataURLStruct</var> be a <a>struct</a> consisting of a
```

Comment:
I think the struct type should be defined outside the algorithm (like "URL record" currently is) as well as its consituents. Then you can have a single step that says "Return a data URL struct whose MIME type is mimeTypeBytes and whose body is body."